### ISSUE TITLE
Uniform Theme for Unrestricted Pages

### ISSUE THREAD

**Issue by B3Chill** [dsiver/SnapTrivia#44]
Will resolve itself after statistics pages are reworked.

**Comment by kchasta2** [comment]
My problem with this one is that I have no idea where the css for the back end pages (Admin - Reviewer pages) is coming from.

**Comment by dsiver** [comment]
I would not attempt to change any CSS in rails_admin for Reviewer or Admin.  Customizing anything to do with this gem is not worth it. You typically have to override its classes to make even subtle changes. Please do not go into these areas in code. It is very sensitive and easy to break.

The dashboard is limited access anyway and the different theme doesn't matter and it's probably a good thing because the current user would know they are in a restricted place.

**Comment by B3Chill** [comment]
Ok all pages that the user see's. It was late sorry for the error.

**Comment by kchasta2** [comment]
Can you be more specific about which pages, please? 

For example, I think the page where the user can update their profile is tied up with the same CSS as the Admin / Reviewer pages.  The same goes for the Inbox and Messaging page. (I think - please correct me if I'm wrong.)

The Rules page and the Game page already use the same CSS.  Which other pages are viewable by a user?

**Comment by B3Chill** [comment]
Maye this one should be removed, the only page that doesn't match the
bootstrap look is the rules page but I guess that doesn't matter.

**Comment by dsiver** [comment]
Updating profile is part of Devise.

The reason rails_admin looks similar to some pages is because it uses
Bootstrap.

I did not style Messages because I'm horrible at it. I created this part
from scratch.

**Comment by dsiver** [comment]
These aren't even areas of the project we should be concerned about at this
time.

What's the main thing that has caused unsatisfactory milestones for the
past three meetings?

Game play.

Function over form, people. We need to be all hands on deck with game play.

Thanks,

David Siver

**Comment by B3Chill** [comment]
You guys asked me to post a list of things that needed to be done. I didn't
rate them with priorities, I just added them. If they don't need to be on
the list close them. If I forgot to add something add it. I agree that we
need to work through them in the highest priority order but I figured that
we are all adults and can figure out the order. As for this issue I was
adding clarification, if this issue bothers anyone close it.
